cds.lib

```diff
@@ -3,3 +3,6 @@ DEFINE     basic        $CDS_HOME/tools/dfII/etc/cdslib/basic
 DEFINE     sky130_fd_pr_main ./libs/sky130_fd_pr_main
 DEFINE     tvip_example tvip_example
 DEFINE     Bandgap_reference    ./Bandgap_reference
+DEFINE     2#2e4GHz_LO ./2#2e4GHz_LO
```

Comment:
The cds.lib references the library with the `#` instead of the ` _` . This is probably not intended since you will likely want to delete the folder `2#2e4GHz_LO` in favor of keeping the _